Source/Public/Get-BicepConfig.ps1

```diff
@@ -0,0 +1,52 @@
+function Get-BicepConfig {
+    [CmdletBinding(DefaultParameterSetName='Default')]
+    param (
+        [Parameter(ParameterSetName='PathOnly', Mandatory)]
+        [Parameter(ParameterSetName='PathLocal', Mandatory)]
+        [Parameter(ParameterSetName='PathMerged', Mandatory)]
+        [ValidateNotNullOrEmpty()]
+        [ValidateScript( { Test-Path $_ } , ErrorMessage = "File not found")]
+        [string]$Path,
+
+        [Parameter(ParameterSetName='PathLocal', Mandatory)]
+        [Switch]$Local,
+
+        [Parameter(ParameterSetName='PathMerged', Mandatory)]
+        [Switch]$Merged,
+
+        [Parameter(ParameterSetName='Default')]
+        [Switch]$Default
+    )
+    begin {
+        # Check if a newer version of the module is published
+        if (-not $Script:ModuleVersionChecked) {
+            TestModuleVersion
+        }
+    }
+
+    process {
+        if ($Path) {
+            $BicepFile = Resolve-Path -Path $Path
+        }
+        $Params = @{
+            Scope = 'Merged'
+        }
+        switch -Wildcard ($PSCmdlet.ParameterSetName) {
+            'PathLocal' {
+                $Params['Scope'] = 'Local'
+            }
+            'Path*' {
+                $Params['Path'] = $BicepFile
+            }
+            'Default' {
+                $Params['Scope'] = 'Default'
+            }
+        }
+        try {
+            Get-BicepNetConfig @Params -ErrorAction 'Stop'
+        }
+        catch [System.ArgumentException] {
```

Comment:
I _think_ this is here to cover some of the described scenarios... But for future reference and reasoning I think you should either:
- Leave a comment why this catch doesn't handle the error but does something else
- If needed, add a generic catch for errors not in ArgumentException.

Reply:
We are catching the error when no bicepconfig if found and instead of exposing that we return the default config.

Reply:
I've added a comment to explain the scenario